crates/fuzzing/src/generators/table_ops.rs

```diff
@@ -132,17 +157,21 @@ mod tests {
 
         let expected = r#"
 (module
-  (import "" "gc" (func))
-  (table $table 10 externref)
-  (func (export "run") (param externref externref)
-    (call 0)
-    (drop (table.get $table (i32.const 0)))
-    (table.set $table (i32.const 1) (local.get 2))
-    (table.set $table (i32.const 3) (table.get $table (i32.const 4)))
-  )
-)
+  (type (;0;) (func (param externref externref)))
+  (import "" "gc" (func (;0;) (type 0)))
+  (func (;1;) (type 0) (param externref externref)
+    call 0
+    drop
+    table.get 0
+    table.set 1
+    local.get 2
+    table.set 3
+    table.get 4)
+  (table (;0;) 10 externref)
+  (export "run" (func 0)))
```

Comment:
cc @fitzgen I'm like .. 99% sure this new output is doing something different than your original expected output but I'm not exactly sure how to use the library to produce a concise copy. 

(other than the order of sections which I followed https://webassembly.github.io/spec/core/binary/modules.html#binary-module for per the docs)

Reply:
See my comments below :)

Reply:
oh awesome you just reviewed I'll resolve this one and use your comments